src/main/java/gyro/aws/autoscaling/AutoScalingCustomizedLoadMetricSpecification.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2020, Brightspot.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.aws.autoscaling;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import gyro.aws.Copyable;
+import gyro.core.resource.Diffable;
+import gyro.core.resource.Updatable;
+import gyro.core.validation.Required;
+import gyro.core.validation.ValidStrings;
+import software.amazon.awssdk.services.autoscalingplans.model.CustomizedLoadMetricSpecification;
+
+public class AutoScalingCustomizedLoadMetricSpecification extends Diffable
+    implements Copyable<CustomizedLoadMetricSpecification> {
+
+    List<AutoScalingMetricDimension> dimensions;
+    private String metricName;
```

Comment:
Update the field, getter/setter and corresponding examples
```suggestion
    private String name;
```